docs/source/command_line.rst

```diff
@@ -344,10 +344,10 @@ Here are some more useful flags:
 .. _config-file-flag:
 
 - ``--config-file CONFIG_FILE`` causes configuration settings to be
-  read from the given file.  By default settings are read from ``mypy.ini``
-  in the current directory.  Settings override mypy's built-in defaults
-  and command line flags can override settings.  See :ref:`config-file`
-  for the syntax of configuration files.
+  read from the given file. By default settings are read from ``mypy.ini``,
+  ``setup.cfg``, or ``tox.ini`` in the current directory. Settings override
```

Comment:
You still mention tox.ini here (and several other places). Also you should clarify that it stops at the first file it finds (here and in the next chapter).